.github/workflows/ci.yml

```diff
@@ -130,7 +137,7 @@ jobs:
         -DF3D_LINUX_GENERATE_MAN=ON
         -DF3D_LINUX_INSTALL_DEFAULT_CONFIGURATION_FILE_IN_PREFIX=ON
         -DF3D_MACOS_BUNDLE=${{ matrix.bundle_label == 'bundle' && 'ON' || 'OFF' }}
-        -DF3D_MODULE_EXTERNAL_RENDERING=ON
+        -DF3D_MODULE_EXTERNAL_RENDERING=${{ matrix.egl_label == 'egl' && 'OFF' || 'ON' }}
```

Comment:
Why would you disable external context with egl?

Reply:
the external rendering testing does not work  at all with EGL, and this is expected.

Reply:
Why is it expected? The external feature means the context is external so there is no use of GLX or EGL directly. It's agnostic of the method used by the caller.
If the tests are not working because QT and GLFW cannot be used, I got it, but the tests themselves should be disabled while the module should be built.  
What happens if the user creates his own EGL context and wants to use libf3d with an external context?

Reply:
Doable but there is no control for egl en f3d code yet. That was the easiest way. I will add egl check and specific code path for egl in f3d later.

Reply:
The external feature of course builds fine with egl, only the test with glfw et qt are not working